platform-sdk/swirlds-common/src/main/java/com/swirlds/common/merkle/impl/internal/AbstractMerkleNode.java

```diff
@@ -105,7 +105,8 @@ public void setRoute(final MerkleRoute route) {
             // allowed to proceed, a node in multiple trees would have the incorrect route in at least one of those
             // trees. Instead of moving the node, make a copy and move the copy.
             throw new MerkleRouteException("Routes can not be set unless the reservation count is 0 or 1. (type = "
-                    + this.getClass().getName() + ", reservation count = " + getReservationCount() + ")");
+                    + this.getClass().getName() + ", reservation count = " + getReservationCount() + ", route = "
+                    + getRoute() + ", new route = " + route + ")");
```

Comment:
This change looks like https://github.com/hashgraph/hedera-services/pull/14858 , which is supposed to be ported separately. I don't believe this change belongs to this PR (#14900).